reference/component-versions.md

```diff
@@ -5,6 +5,71 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated, they are marked with `n/a` below.
 
+## 1.23.0
+
+### Charms
+
+#### Ubuntu 22.04
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.23/stable` | 449 |
+| `ams-node-controller` (Deprecated) | `1.23/stable` | 457 |
+| `coturn` | `1.23/stable` | 451 |
+| `ams` | `1.23/stable` | 481 |
+| `anbox-stream-gateway` | `1.23/stable` | 467 |
+| `ams-lxd` | `1.23/stable` | 454 |
+| `anbox-stream-agent` | `1.23/stable` | 462 |
+| `nats` (For AMD64) | `latest/stable` | 9 |
+| `nats` (For ARM64) | `latest/stable` | 11 |
+
+#### Ubuntu 20.04 (Deprecated)
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.23/stable` | 448 |
+| `ams-node-controller` (Deprecated) | `1.23/stable` | 456 |
+| `coturn` | `1.23/stable` | 450 |
+| `ams` | `1.23/stable` | 480 |
+| `anbox-stream-gateway` | `1.23/stable` | 466 |
+| `ams-lxd` | `1.23/stable` | 454 |
+| `anbox-stream-agent` | `1.23/stable` | 461 |
+| `nats` (For AMD64) | `latest/stable` | 10 |
```

Comment:
Is it worthwhile to collapse the two in the same row as the following ?

```
| `nats` (For ARM64) | `latest/stable` | 10(AMD64), 12 (ARM64) |
```

Reply:
Good idea, let me do that